mobsfy_AVD.py

```diff
@@ -214,7 +219,7 @@ def main():
         print_log("Emulator INI already exists")
     else:
         shutil.copyfile(emulator_ini, new_emulator_ini)
-    if os.path.isdir(avd_path):
+    if os.path.isdir(new_emulator_avd):
```

Comment:
We need to overwrite any files that we copy to `sdk` and `avd` directory. Also please give a graceful message that we will overwrite existing files. The overwrite is necessary as we may release avd image updates in future.